src/components/home/modals/LeaseInput.vue

```diff
@@ -336,6 +336,9 @@ export default {
             return browser.numberFormatter(num)
         },
         addRecipientList: function() {
+            if (this.amount !== BigNumber(this.amount).toNumber().toString()) {
+                alert('Warning :The amount  is too large. ' + this.amount + ' will round to ' + BigNumber(this.amount).toNumber().toString() + '.')
+            }
```

Comment:
Do not add warning in `addRecipientList`. Use new function to handle amount warning.

Reply:
The text should be "... round to 100000000".
<img width="450" alt="screenshot 2019-01-31 at 4 54 43 pm" src="https://user-images.githubusercontent.com/41148850/52042985-812d2200-2579-11e9-82f4-fad7f1f93c6a.png">